.gitignore

```diff
@@ -56,4 +60,10 @@ buck-out/
 *.jsbundle
 
 # CocoaPods
-/ios/Pods/
+**/ios/Pods/
+*Podfile.lock
+
+**/dist
+**/.vscode
+**/.log
+**/.env
```

Comment:
This file should not be here as it was part of the other pull request.

Reply:
I made this commit after I worked on the `.gitignore` file.